src/ert/parsing/lark_parser.py

```diff
@@ -362,7 +362,9 @@ def _handle_includes(
                         filename=config_file,
                     ).set_context(error_context)
                 )
-                continue
+
+                args = args[0:1]
+                # continue
```

Comment:
why continue commented?

Reply:
Oops, should be removed instead. Omiting the `continue` will make it just ignore superfluous tokens, but still give an error about them.